### ISSUE TITLE
Exception when restoring state in Sonos binding

### ISSUE THREAD

**Issue by kaikreuzer** [openhab/openhab-addons#32]
When calling restoreAll, I get the following exception:

```

15:12:50.068 DEBUG o.e.s.c.t.i.ThingManager[:161] - Delegating command 'ON' for item 'sonos_zoneplayer_RINCON_B8E93780863601400_restoreall' to handler for channel 'sonos:zoneplayer:RINCON_B8E93780863601400:restoreall'
15:12:50.072 ERROR o.e.s.c.t.i.ThingManager[:166] - Exception occured while calling handler: null
java.lang.NullPointerException: null
    at java.math.BigDecimal.<init>(BigDecimal.java:739)
    at org.eclipse.smarthome.core.library.types.DecimalType.<init>(DecimalType.java:50)
    at org.eclipse.smarthome.core.library.types.DecimalType.valueOf(DecimalType.java:58)
    at org.openhab.binding.sonos.handler.ZonePlayerHandler.restoreState(ZonePlayerHandler.java:979)
    at org.openhab.binding.sonos.handler.ZonePlayerHandler.restoreAllPlayerState(ZonePlayerHandler.java:271)
    at org.openhab.binding.sonos.handler.ZonePlayerHandler.handleCommand(ZonePlayerHandler.java:233)
    at org.eclipse.smarthome.core.thing.internal.ThingManager.receiveCommand(ThingManager.java:164)
    at org.eclipse.smarthome.core.events.AbstractEventSubscriber.receiveCommand(AbstractEventSubscriber.java:135)
    at org.eclipse.smarthome.core.events.AbstractEventSubscriber.handleEvent(AbstractEventSubscriber.java:105)
    at org.eclipse.equinox.internal.event.EventHandlerWrapper.handleEvent(EventHandlerWrapper.java:197)
    at org.eclipse.equinox.internal.event.EventHandlerTracker.dispatchEvent(EventHandlerTracker.java:197)
1
```

**Comment by kgoderis** [comment]
This should be solved now, cfr #97

**Comment by kaikreuzer** [comment]
True.
